test/libsolidity/syntaxTests/nameAndTypeResolution/013_large_string_literal.sol

```diff
@@ -1,7 +1,6 @@
 contract test {
-    function f() public { var x = "123456789012345678901234567890123"; }
+    function f() public { string memory x = "123456789012345678901234567890123"; }
 }
 // ----
-// Warning: (42-47): Use of the "var" keyword is deprecated.
-// Warning: (42-47): Unused local variable.
-// Warning: (20-88): Function state mutability can be restricted to pure
+// Warning: (42-57): Unused local variable.
```

Comment:
These warnings are not relevant in the scope of this test and can be silenced with ``function f() public pure { string memory x = "123456789012345678901234567890123"; x; }``

Reply:
I'm fine with changing semantics of the test, but I didn't want to change more than this PR was meant for, i.e., this is about replacing `var` keyword with their actual explicit types. I'd love to create a soltest-suite-polish PR that does exactly that, what you'd like to see (too) :)